### PR TITLE
rgw: exit early if rgw_bucket_set_attrs() fails

### PR DESCRIPTION
**Pull request by dengxiafubi**
Check for the return value of rgw_bucket_set_attrs() and exit earlier in case of
errors

Signed-off-by: dengxiafubi <xhwcqupt@sina.com>

### CODE REVIEW

File: src/rgw/rgw_op.cc [hunk 1]
```diff
@@ -4775,6 +4775,11 @@ void RGWDeleteLC::execute()
       }
     }
   op_ret = rgw_bucket_set_attrs(store, s->bucket_info, attrs, &s->bucket_info.objv_tracker);
+  if (op_ret < 0) {
+    dout(0) << "RGWLC::RGWDeleteLC() failed to set attrs on bucket=" << s->bucket.name
```
Comment (theanalyst) [inline src/rgw/rgw_op.cc]: while you're at it, also move from `dout`-> `ldout(s->cct,0)`

**Comment by dengxiafubi** [comment]
@joscollin update

**Comment by dengxiafubi** [comment]
@theanalyst update

File: src/rgw/rgw_op.cc (hunk 1 above)
Comment (joscollin) [inline src/rgw/rgw_op.cc]: @theanalyst Do you mean move the other dout's in execute() too ?

**Comment by yuriw** [comment]
ready for merge

http://pulpito.ceph.com/yuriw-2017-08-21_21:16:27-rgw-wip-yuri-testing4_2017_8_22-distro-basic-smithi/

http://pulpito.ceph.com/yuriw-2017-08-22_14:54:48-rgw-wip-yuri-testing4_2017_8_22-distro-basic-smithi/

**Comment by alfredodeza** [comment]
jenkins test docs

**Pull request merged**